### ISSUE TITLE
Error: no suport windows 8

### ISSUE THREAD

**Issue by gladson** [ekalinin/nodeenv#53]
```File "C:\Users\Gladson\Desktop\Teste\env_nodeenv\lib\site-packages\nodeenv.py",
line 299, in callit % (cmd_desc, proc.returncode))
OSError: Command curl --silent -L http://nodejs.org/dist/node-v-RC1.tar.gz | tar 
xzf - -C "'C:\Users\Gladson\De...este\env_node\src'" failed with error code 255 ```

Have to install all dependencies: 
https://github.com/ekalinin/nodeenv#dependency 

It would be better if there were not so many dependencies.

**Comment by asottile** [comment]
I'm going to poke at this today (trying to get better windows support in https://github.com/pre-commit/pre-commit).  I think eliminating a few of the requirements is pretty straightforward.

**Comment by ekalinin** [comment]
:+1:

**Comment by asottile** [comment]
Some notes for now:

The configuration I'm targeting is `Git Shell` (provided by github tools), though I should be able to get it working with less than that probably.

- #114 eliminates requirements for `egrep` and `sort`
- `curl` (at least in my configuration) has different options than unix.  Should be easy enough to replace with the `urllib` equivalent however and eliminate that binary dependency.
- `tar` (at least in my configuration) seems identical to POSIX tar
- The node binaries are packaged differently for windows (ugh)
 - For nodejs.org they are at `/dist/{version}/node.exe` for x86 and `/dist/{version}/x64/node.exe` for x86_64
 - For iojs.org they are at `/dist/{version}/win-x86` and `/dist/{version}/win-x64`

**Comment by techtonik** [comment]
I believe this now can be moved to #140, because error message have changed since then, and some progress was made on `--prebuilt` binaries.

**Comment by ekalinin** [comment]
Will be fixed in #140
